serviceUsage.html

```diff
@@ -345,8 +362,94 @@ <h2>
                 <td>When specified message protocol is not loaded</td>
             </tr>
         </table>
-
-    </section>
+		<h2>
+            <a id="examples" class="anchor" href="#examples" aria-hidden="true">
+                <span aria-hidden="true" class="octicon octicon-link"></span>
+            </a>
+        </h2>
+		<h2> Lookups </h2>
+		<p> Presently, to generate Eiffel messages, Eiffel REMREM-Generate is being used. This process requires input mapping exactly one-to-one to Eiffel message Schema.</p>
+		<p> The "Link" field of Eiffel Message requires UUIDs of earlier sent events to link to. Passing these UUID's each time is a little bit difficult, some times user lookup the ER and manually place those IDs in REMReM Generate semi-structured event. In order to resolve this , we need to provide LOOKUP's for the Link's field ,which will have eventType and Properties. eventType will contain type of event.</p>
+		<p> Properties will contain any field of event mentioned in eventType or RepoUri or artifact identity (Purl identity including version) to Eiffel,
+			which will make sure it is linked to the correct events. The lookups are used in filling the information for links, which will help users to know what event they want to query with the parameters so that they are automatically fetched and linked by Generate instead of the users adding them as a mandatory param when generating events.
+		</p>
+		<p> <em>parameters</em> specified should be enough to replace the listed event id as input will help to prepare the Query to ER and Lookup the ER data.</p>
+		<p><h6>Example:</h6>
+			As a User, I want to place the LINK type ARTIFACT to Event ID of EiffelArtifactCreatedEvent from Event Repository.
+			The user passes the below template information to REMReM Generate, REMReM Generate will Query to configured Event Repository and place the event ID in LINK's and generate the Eiffel message.
+		</p>
+		<h5> Templates for LOOKUPS : </h5>
+		<pre>
+		"links":[
+					{
+			"type": "ARTIFACT",
+			"%lookup%": {
+			"eventType": "EiffelArtifactCreatedEvent",
+			"properties": [
+				{ “data.identifier": "some purl identifier defining the artifact" }]
+					}
+				}]
+		</pre>
+		<p><h6>ER Query will look like:</h6></p>
+		<pre>https://host:port/eventrepository/events/?eventType=EiffelArtifactCreatedEvent&data.identifier=some purl identifier defining the artifact</pre>
+		<h5> Examples for Lookups:</h5>
+		<h6> Example 1:</h6>
+		<pre>
+		"links": [
+			"type": “ARTIFACT",​
+			"%lookup%": {​
+			"eventType": "EiffelArtifactCreatedEvent",​
+			"properties": [
+					{​“data.identity": "pkg:maven/com.ericsson.eiffel/testartifact@0.0.21?type=xml&classifier=test"}
```

Comment:
replace the namespace from com.ericsson.eiffel to test.
Also remove the queryparams from the URL